vc_agent

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
+make_credentials () {
+  mkdir $VC_HOME/.aws
```

Comment:
Is this the right directory?  I think it looks for the home of the user that is running the script, which would be root, so `/root`, but I haven't looked at the docs that closely.